### PR TITLE
Adding new robot by MIP robotics the MIP junior 300

### PR DESCRIPTION
**Pull request by MIP-robotics**
Hello,

I hope this new robot can be added to the ROs robot store.

Thank you,
MIP robotics

### CODE REVIEW

File: _posts/2020-08-28-mip-robotics-junior-300.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+- 'ROS-industrial'
+- 'arm'
+- 'robotic arm'
+- 'collaborative'
+- 'cobot'
+- 'industrial'
+- 'pick and place'
+twitter_text: Checkout the MIP robotics Junior 300
+website: https://www.mip-robotics.com/
+wiki_homepage: https://github.com/MIPRobotics/ROS_robotics_simulation
```
Comment (tfoote) [inline _posts/2020-08-28-mip-robotics-junior-300.md]: This should be a link to a page on the ROS wiki with developer documentation for how to use the robot with the ROS interface. https://github.com/ros-infrastructure/robots.ros.org/blob/gh-pages/contributing.md#what-content-should-be-on-the-site

The page that this currently links to has no content beyond the title.

**Review by tfoote — CHANGES_REQUESTED** [review]

File: _posts/2020-08-28-mip-robotics-junior-300.md (hunk 1 above)
Comment (MIP-robotics) [reply _posts/2020-08-28-mip-robotics-junior-300.md]: Yes, this repo will have the instructions. 
Currently, only the scaffolding was generated. 
We expect the repo to be full by end of next week.

Comment (tfoote) [reply _posts/2020-08-28-mip-robotics-junior-300.md]: Please read the instructions. They specifically talk about having a ROS wiki page with the developer overview about how to get up and running. This helps with discovery through a consistent user experience. The wiki page can point to more detailed documentation on github or elsewhere. 

Please note that the link you provide to the bare repo doesn't have any documentation, you have to click into other files to find any significant documentation.

Comment (MIP-robotics) [reply _posts/2020-08-28-mip-robotics-junior-300.md]: Wiki created and link updated

**Comment by tfoote** [comment]
Please either embed the installation instructions in the wiki page or link directly to them. You have only one link that takes you to a top level of the repository with a Readme that has no links to developer documentation. The user is required to then go browsing through various readme files to find the right ones. YOu should at least link to the installation docs and the operations documents.

**Review by tfoote — APPROVED** [review]
Thanks for filling that in. I made the link on the wiki a little bit clearer too.

**Pull request merged**